### DISCUSSION TITLE
Github Docs: CNAME was incorrectly written as ANAME

### DISCUSSION THREAD

**Discussion by CodeZhangBorui** [community/community#20434] [Other Feature Feedback, Questions, & Ideas] [answered]
Github Docs: CNAME was incorrectly written as ANAME

It seems that there is a wrong word on [Github Pages - Manage a custom domain for your github pages site - Configuring an apex domain](https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain) in all of languages.

The website says:

> To set up an apex domain, such as `example.com`, you must configure a CNAME file in your GitHub Pages repository and at least one `ALIAS`, ***`ANAME`***, or `A` record with your DNS provider.

I think the word `ANAME` should be `CNAME`.

**Comment by FahdSeddik** [comment]
Hi @CodeZhangBorui ,
Yes, you are right.

**Comment by y9su** [comment, marked as answer]
You might be correct! Submit the bug at- https://support.github.com/contact/bug-report

Please accept my answer if I helped out.

**Reply by CodeZhangBorui** [reply]
I have submited a report through the Github Bug report!
